### DISCUSSION TITLE
Small typo in website's documentation: RFC 3336 -> RFC 3339

### DISCUSSION THREAD

**Discussion by ludoux** [kimai/kimai#3690] [General]
https://www.kimai.org/documentation/rest-api.html

> When POSTing or PATCHing timesheet records, you MUST use the HTML5 format (see [RFC 3336](https://tools.ietf.org/html/rfc3339) as well)

the text RFC 333**6** should be RFC 333**9** .(the hyperlink url is current)

![图片](https://user-images.githubusercontent.com/13460812/208280806-2a1bcbcd-5fbf-4cdf-8132-774b9355873d.png)

It's just a small typo on website so I submit it here instead of issues :)

**Comment by ludoux** [comment]
the hyperlink url is ~~current~~ correct

**Reply by kevinpapst** [reply]
Wow, you have a hawk eye 😁  Thanks for spotting & sharing 👍 Fixed.
